source/view.c

```diff
@@ -2169,7 +2166,30 @@ void rofi_view_cleanup(void) { proxy->cleanup(); }
 
 void rofi_view_hide(void) { proxy->hide(); }
 
-void rofi_view_reload(void) { proxy->reload(); }
+static gboolean _rofi_view_reload_widget(void *data) {
+  widget *w = (widget *)data;
+  if (w) {
+    widget_queue_redraw(WIDGET(w));
+  }
+  widget_unref(w);
```

Comment:
No need to take risks :)

```suggestion
  if (w) {
    widget_queue_redraw(WIDGET(w));
    widget_unref(w);
  }
```